docs/data-cookbooks/run-tasks-in-lambda-or-docker.md

```diff
@@ -0,0 +1,99 @@
+---
+id: run-tasks-in-lambda-or-docker
+title: Run Step Function Tasks in Lambda or Docker
+hide_title: true
+---
+
+# Running Step Function Tasks in AWS Lambda or Docker
+
+## Overview
+
+[AWS Step Function Tasks](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-tasks.html) can run tasks on [AWS Lambda](https://aws.amazon.com/lambda/) or on [AWS Elastic Container Service (ECS)](https://aws.amazon.com/ecs/) as a Docker container.
+
+Lambda provides serverless architecture, providing the best option for minimizing cost and server management. ECS provides the flexibility to execute arbitrary code on any AWS EC2 instance type. 
```

Comment:
I know it's specified in greater detail below, but should we briefly mention that ECS gives you more resources? Something like "..., allowing you run tasks that need more resources"

Reply:
Added a bit here https://github.com/nasa/cumulus/pull/902/commits/fd330bcc774c347bc53fcfdc576cb3f2be85ebdc let me know what you think